osu.Game/Screens/OsuScreen.cs

```diff
@@ -52,6 +54,14 @@ public abstract class OsuScreen : Screen, IOsuScreen, IKeyBindingHandler<GlobalA
 
         protected new OsuGameBase Game => base.Game as OsuGameBase;
 
+        /// <summary>
+        /// The <see cref="UserStatus"/> to set the user's status automatically to when this screen is entered / resumed.
+        /// Note that the user status won't be automatically set if :
+        /// <para>- <see cref="ScreenStatus"/> is overriden and returns null</para>
+        /// <para>- The current <see cref="UserStatus"/> is <see cref="UserStatusDoNotDisturb"/> or <see cref="UserStatusOffline"/></para>
+        /// </summary>
+        protected virtual UserStatus ScreenStatus => new UserStatusOnline();
```

Comment:
There should be a pathway for a screen to update the status while it's active.

Reply:
A settable property should be good